src/components/hello/index.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div>
+        hello，欢迎你学习vue。
+    </div>
+</template>
+
+<script>
+    import Parent from "../parent.vue";
+    import Child from "../child.vue";
+    import mixin from "../../mixin"
+
+    export default {
+        name: "index",
+        data(){
+            return {
+                message:'waowao'
+            }
+        },
```

Comment:
Codacy has a fix for the issue: [Replace `········` with `····`](https://app.codacy.com/gh/NickName039X/vue-demo/pullRequest?prid=9343708)
```suggestion
    },
```